test/test_file.cc

```diff
@@ -0,0 +1,45 @@
+/* test_path.cc - Test path functionality
```

Comment:
test_path.cc -> test_file.cc
Sorry did not see this before.

Reply:
```suggestion
/* test_file.cc - Test file functionality
```